docs/template_plugin/src/CMakeLists.txt

```diff
@@ -21,6 +21,7 @@ target_include_directories(${TARGET_NAME} PRIVATE
 
 # link common Inference Engine libraries
 target_link_libraries(${TARGET_NAME} PRIVATE
+    pugixml # for dummy implementation of Export / ImportNetwork flow
```

Comment:
Why do you need to have `pugixml` dependency? Looks like serializer and reader already contain pugi.

Reply:
looks like you are right.